widget/label.go

```diff
@@ -82,7 +82,7 @@ func (l *Label) object() fyne.Widget {
 func (l *Label) CreateRenderer() fyne.WidgetRenderer {
 	l.ExtendBaseWidget(l)
 	l.provider = newTextProvider(l.Text, l)
-	l.provider.Hidden = l.Hidden
```

Comment:
If develop was merged in I think this goes away :)

Reply:
Yup, thanks!